packages/snap/src/generate-snapshots/build-base-files.ts

```diff
@@ -3,7 +3,12 @@ import {generateSnapshotFilename, generateData} from './filename-utils';
 import { IRegistry } from '@ui-autotools/registry';
 import path from 'path';
 
-export const buildBaseFiles = (projectPath: string, Registry: IRegistry, fs: any) => {
+export interface IFileInfo {
+  filename: string;
```

Comment:
I'd rename `filename` to `basename` to make it more clear that it doesn't include the extension.